cartographer_ros/cmake/FindEigen.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Cartographer Authors
```

Comment:
Sure you need this file again?
I thought cartographer_ros/CMakeLists calls `include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")` cartographer/cmake/functions.cmake which adds cartographer/cmake/modules

Same for cartographer_rviz